package.json

```diff
@@ -17,7 +18,8 @@
     "app-store": "yarn app-store-cli cli",
     "create-app": "yarn app-store create",
     "edit-app": "yarn app-store edit",
-    "publish-embed": "yarn workspaces foreach --from=\"@calcom/embed*\" npm publish --access=public",
+    "withHostedCalcomEnv": "NEXT_PUBLIC_EMBED_LIB_URL='https://app.cal.com/embed/embed.js' NEXT_PUBLIC_WEBAPP_URL='https://app.cal.com' yarn",
+    "publish-embed": "yarn withHostedCalcomEnv workspace @calcom/embed-core build && yarn withHostedCalcomEnv workspace @calcom/embed-snippet build && yarn workspaces foreach --from=\"@calcom/embed*\" npm publish --access=public",
```

Comment:
embed-core and embed-snippet are the dependencies for embed-react and any new framework specific package(that might be added). So, build them first.